server/notification-providers/46elks.js

```diff
@@ -0,0 +1,52 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class Elks extends NotificationProvider {
+    name = "Elks";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+
+        try {
+
+            // API credentials
+            const username = notification.elksUsername;
+            const password = notification.elksAuthToken;
+            const authKey = Buffer.from(username + ":" + password).toString("base64");
+
+            // Set the SMS endpoint
+            const url = "https://api.46elks.com/a1/sms";
+
+            // Request data object
```

Comment:
Please remove all the comments which are self-explanatory. Comments should be used only where they explain hard to reason about code.